.github/workflows/ci.yml

```
@@ -146,7 +146,15 @@ jobs:
             env:
                 DOWNSTREAM_PROJECT: ${{ matrix.downstream_project }}
             run: |
-                git clone "https://github.com/inducer/$DOWNSTREAM_PROJECT.git"
+                curl -L -O -k https://gitlab.tiker.net/inducer/ci-support/raw/main/ci-support.sh
+                . ./ci-support.sh
+
+                echo "$GITHUB_HEAD_REF"
+                if [[ "$DOWNSTREAM_PROJECT" = "grudge" ]] && [[ "$GITHUB_HEAD_REF" = "kernel_callables_v3-edit2" ]]; then
```

Thanks for the PR!
Probably
```suggestion
                if [[ "$DOWNSTREAM_PROJECT" = "grudge" ]] && [[ "$GITHUB_BASE_REF" = "kernel_callables_v3-edit2" ]]; then
```

https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

---

Thanks for taking a look! `$GITHUB_BASE_REF` would stop working as soon as it's merged to kcv3. f9021ca only works *once* merged. (A version of) 093e1cf hopefully works in both scenarios.

---

But for other PRs that are also directed towards `kernel_callables_v3-edit2`, this will fallback to inducer/grudge right? or am I missing something?

---

Oh whoops. Fair point.

---

210858c

---

Thank you!